### PR TITLE
Move CORTX_CORT_ARCHITECTURE.rst from Cortx to Cortx-Motr repo.

### PR DESCRIPTION
**Pull request by huanghua78**
This doc is moved from CORTX to CORTX-Motr.

Signed-off-by: Hua Huang <hua.huang@seagate.com>

### CODE REVIEW

File: doc/CORTX_MOTR_ARCHITECTURE.rst [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+
+
+=================
+**What is CORTX?**
+=================
+
+
+-  CORTX core is a component of the new cloud software stack developed by Seagate
+
+-  CORTX provides basic functionality through its clovis interface:
```
Comment (imvenkip) [inline doc/CORTX_MOTR_ARCHITECTURE.rst:43]: Need to remove clovis string.

Comment (huanghua78) [reply doc/CORTX_MOTR_ARCHITECTURE.rst:43]: You are right.

Original doc is needed to edit the image. Otherwise, it will be time-consuming.

Comment (huanghua78) [reply doc/CORTX_MOTR_ARCHITECTURE.rst:43]: The original doc:

https://seagatetechnology.sharepoint.com/:p:/r/sites/GMSites/edsengineering/_layouts/15/Doc.aspx?sourcedoc=%7B6D890421-A1B3-400F-BDE1-726E5FD1CB01%7D&file=EOS%20core%20architecture%20overview.pptx&action=edit&mobileredirect=true&DefaultItemOpen=1

Comment (huanghua78) [reply doc/CORTX_MOTR_ARCHITECTURE.rst:43]: @johnbent , @nikitadanilov do you think we need to update all the images in the doc to reflect the name changes, e.g.  from Clovis to Motr client?

Comment (huanghua78) [reply doc/CORTX_MOTR_ARCHITECTURE.rst]: Anyway, it is fixed now.

Comment (huanghua78) [reply doc/CORTX_MOTR_ARCHITECTURE.rst]: @imvenkip would you like to take another look?

Comment (imvenkip) [reply doc/CORTX_MOTR_ARCHITECTURE.rst]: Sure.

Comment (imvenkip) [reply doc/CORTX_MOTR_ARCHITECTURE.rst]: Looks good to me.

**Review by VenkyOS — APPROVED** [review]
Approved. It looks fine.

**Pull request merged**